design/resources.go

```diff
@@ -199,6 +199,25 @@ var _ = a.Resource("user", func() {
 
 })
 
+var _ = a.Resource("identity", func() {
+	a.BasePath("/identities")
+
+	a.Action("list", func() {
+		a.Security("jwt")
+		a.Routing(
+			a.GET(""),
+		)
+		a.Description("List all identities.")
+		a.Response(d.OK, func() {
+			a.Media(identityArray)
+		})
+		a.Response(d.BadRequest, func() {
```

Comment:
I am going to merge this now, will send another PR for this after your PR gets merged.